lib/cc/services/github_pull_requests.rb

```diff
@@ -85,19 +85,17 @@ def update_status_pending
   end
 
   def update_status(state, description)
-    if config.update_status
-      params = {
-        state:       state,
-        description: description,
-        target_url:  @payload["details_url"],
-        context:     "codeclimate"
-      }
-      service_post(status_url, params.to_json)
-    end
+    params = {
+      state:       state,
+      description: description,
+      target_url:  @payload["details_url"],
+      context:     "codeclimate"
+    }
+    service_post(status_url, params.to_json)
   end
 
   def add_comment
-    if config.add_comment && !comment_present?
```

Comment:
Okay - we resolved our differences and are no longer relying on the config check that @jyurek removed here - so #55 should be compatible with this PR now